### PR TITLE
View Opportunity Page revamp(#164)

### LINKED ISSUE #164: View Opportunity Page Revamp
Revamping of View Opportunity Page:

1. `Post Opportunity` button should redirect to `Post Opportunity` page of respective opportunity.
2. `Select Opportunity` dropdown should have: 
- Fulltime Jobs
- Hackathons
- Scholarships
- Coding Competitions
- Tech Conferences
- Internships
3. The banner heading should change accordingly.
4. Must be responsive for all screen sizes.

[Follow this design](https://www.figma.com/proto/XKmx5ia2hGjM9iz3VdWABp/girlcodeit?node-id=11%3A62&scaling=min-zoom&page-id=0%3A1)

### PR DESCRIPTION
**Pull request by akshitakanojia**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

Fixed #164 

## Type of change

Please check options that are relevant to your PR.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have pulled the latest changes in the repository
- [x] I have squashed my commits
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by vaishali614** [comment]
@akshitakanojia Your changes don't match with the design. Please try to replicate exactly the same.

Some major visible corrections required -
1. While the opportunities are loading, there should be loader visible or you can keep it the same way it was before i.e. Looking for opportunities if any. (Please check the deployment for more clarity) 
2. Dropdown
3. Cards (You can leave cards for now)

**Comment by vaishali614** [comment]
@akshitakanojia Everything looks good, but please modify navbar too as per design.

File: src/Components/ViewOpportunity/Navbar.js [hunk 1]
```diff
@@ -1,10 +1,20 @@
 import React from 'react';
```
Comment (vaishali614) [inline src/Components/ViewOpportunity/Navbar.js:1]: Please modify Navbar too according to design. Rest of the things look good.

**Review by vaishali614 — CHANGES_REQUESTED** [review]

**Review by vaishali614 — APPROVED** [review]

**Comment by vaishali614** [comment]
@akshitakanojia Changes look good now. Please take the latest pull from upstream to get this PR merged.

**Comment by akshitakanojia** [comment]
@vaishali614 I'm getting merge conflicts while rebasing though I've already solved merge conflicts here.

**Pull request merged**